src/Components/Web/src/Forms/ModelBinding/CascadingModelBinder.cs

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection.Metadata;
+using Microsoft.AspNetCore.Components.Forms.ModelBinding;
+using Microsoft.AspNetCore.Components.Rendering;
+
+namespace Microsoft.AspNetCore.Components.Forms;
+
+/// <summary>
+/// Defines the binding context for data bound from external sources.
+/// </summary>
+public sealed class CascadingModelBinder : ICascadingValueSupplier, IComponent
+{
+    private SupplyParameterFromFormValueProvider? _cascadingValueSupplier;
+    private RenderHandle _handle;
+    private bool _hasPendingQueuedRender;
+
+    /// <summary>
+    /// The binding context name.
+    /// </summary>
+    [Parameter] public string Name { get; set; } = "";
+
+    /// <summary>
+    /// Specifies the content to be rendered inside this <see cref="CascadingModelBinder"/>.
+    /// </summary>
+    [Parameter] public RenderFragment<ModelBindingContext> ChildContent { get; set; } = default!;
+
+    [CascadingParameter] private ModelBindingContext? ParentContext { get; set; }
+
+    [Inject] internal NavigationManager Navigation { get; set; } = null!;
```

Comment:
This can be removed

Reply:
Tried, but it's still used as part of generating the mapping context ID.